styles.css

```diff
@@ -203,3 +208,87 @@ h3 {
   background-position: right;
   background-repeat: no-repeat;
 }
+
+.contact {
+  padding: 24px;
+}
+
+.contact h1 {
+  font-size: 32px;
+  text-align: center;
+  padding-left: 0;
+}
+
+.contact p {
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input {
+  height: 48px;
+  width: 100%;
+  margin: 12px 0;
+}
+
+#message {
+  width: 100%;
+  height: 148px;
+  margin: 12px 0;
+}
+
+#btn {
+  margin: 30px 0;
+}
+
+.email-icon {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.line hr {
+  border: 1px solid #3a3837;
+}
+
+.icon {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px 50px 10px;
+  list-style: none;
+  width: 50%;
+  margin: 0;
+}
+
+.email {
+  color: #fff;
+  text-decoration: none;
+  font-size: 20px;
+  line-height: 28px;
+  padding-bottom: 40px;
+}
+
+.btn:hover {
+  background: #7d73fc;
+  color: #fff;
+  font-size: 12px;
+}
```

Comment:
Great job team 🤝

- [ ] [Optional] You may as well add the `cursor: pointer` property to the buttons to add the UX experience.